include/openmc/boundary_condition.h

```diff
@@ -16,14 +19,44 @@ class Surface;
 class BoundaryCondition {
 public:
   //! Perform tracking operations for a particle that strikes the boundary.
+  //! Handle the boundary albedo and modify the incident particle's weight.
   //! \param p The particle that struck the boundary.  This class is not meant
   //!   to directly modify anything about the particle, but it will do so
   //!   indirectly by calling the particle's appropriate cross_*_bc function.
   //! \param surf The specific surface on the boundary the particle struck.
-  virtual void handle_particle(Particle& p, const Surface& surf) const = 0;
+  virtual void handle_particle(Particle& p, const Surface& surf) const
```

Comment:
I do not like the approach of having a function named `handle_particle` only handle albedo. We should make this a non-virtual method called `handle_albedo`, then leave `handle_particle` as `=0`'d here.

Reply:
Fixed the PR @gridley ! That should do it.